src/index.ts

```diff
@@ -79,6 +79,12 @@ const useHotkeys = (
         return;
       }
 
+      // chrome autocomplete triggers 'keydown' event but event.key will be
+      // undefined. See https://bugs.chromium.org/p/chromium/issues/detail?id=581537
+      if (!Boolean(event.key)) {
```

Comment:
Could we check `event.key === undefined` instead? Potentially more readable. Might also be worth moving this to the very top of the `onKeydown` handler so that we bail immediately if this occurs. Thanks for spotting this and taking the time to fix, and sorry for the delay in getting back to you!

Reply:
Sure :)